autoload/EasyMotion.vim

```diff
@@ -53,7 +53,6 @@ function! EasyMotion#init()
     let s:EasyMotion_is_cancelled = 0
     " 0 -> Success
     " 1 -> Cancel
-    let g:EasyMotion_ignore_exception = 0
```

Comment:
It's my fault because I didn't write documentation for this variable, but this variable is used by https://github.com/haya14busa/vim-easyoperator-line.
Could you please reverting this line and check this var when printing exception?
If the verbose option is set, it's ok to suppress the exception regardless of this variable.